CHANGELOG.md

```diff
@@ -1,8 +1,35 @@
 # Change Log
 
+<a name="v1.0.0"></a>
+
+## [v1.0.0](https://github.com/auth0/go-auth0/tree/v1.0.0) (2023-07-27)
+
+[Full Changelog](https://github.com/auth0/go-auth0/compare/v0.17.2..v1.0.0)
+
+### Added
+
+- Generate safe getters for `map[string]interface{}` types ([#205](https://github.com/auth0/go-auth0/pull/205))
+- Expose configuration of retry strategy ([#216](https://github.com/auth0/go-auth0/pull/216))
+- Add Authentication Client ([#222](https://github.com/auth0/go-auth0/pull/222), [#227](https://github.com/auth0/go-auth0/pull/227), [#226](https://github.com/auth0/go-auth0/pull/226), [#229](https://github.com/auth0/go-auth0/pull/229), [#232](https://github.com/auth0/go-auth0/pull/232))
+- Validate `org_name` claim in ID token ([#246](https://github.com/auth0/go-auth0/pull/246))
+- Added support for `azure_cs` and `ms365` email providers ([#247](https://github.com/auth0/go-auth0/pull/247))
+- Added support for `OIDCLogoutPrompt` setting on tenant ([#249](https://github.com/auth0/go-auth0/pull/249))
+- Added support for `DecryptionKey` on SAML Connection Options ([#251](https://github.com/auth0/go-auth0/pull/251))
+
+### Breaking Changes
+
+- Improve typing of Client Addons ([#208](https://github.com/auth0/go-auth0/pull/208), [#228](https://github.com/auth0/go-auth0/pull/228))
+- Accept a `context.Context` as the first argument for all methods ([#212](https://github.com/auth0/go-auth0/pull/212))
+- Refactor Management struct to avoid chaining issue ([#214](https://github.com/auth0/go-auth0/pull/214))
+- Removed deprecated `EmailManager` ([#218](https://github.com/auth0/go-auth0/pull/218))
```

Comment:
```suggestion
- Remove deprecated `EmailManager` ([#218](https://github.com/auth0/go-auth0/pull/218))
```

Reply:
The changelog up til now mostly uses the past tense rather than present, personally I think that past does make sense and really the use of `Added` is superfluous give the `Added` heading.

What do you think to dropping `Added` from individual entries and sticking with `Removed` to maintain consistency with the whole changelod?

Reply:
Makes sense 👍🏻